package.json

```diff
@@ -56,6 +58,7 @@
     "karma": "^5.2.3",
     "karma-chrome-launcher": "^3.1.0",
     "karma-coverage": "^2.0.3",
+    "karma-debug-launcher": "0.0.2",
```

Comment:
Nice - I like how you identified this need and "just" published a library for it. 
Any reason this needs to be pinned to `0.0.2`?

Reply:
Nevermind, I saw the later commit...